.changes/v2.21.0/579-improvements.md

```diff
@@ -0,0 +1,2 @@
+* External Network V2 now supports IP Spaces on VCD 10.4.1+ with new fields `UsingIpSpace` and
```

Comment:
What about specifying exact type name you're referring with "External Network V2" too?

Reply:
Fixed